src/Slices/DesiredState/UI/Page.tsx

```diff
@@ -34,76 +41,106 @@ export const Page: React.FC = () => {
     route: "DesiredState",
     keys: { date: "DateRange", version: "IntRange" },
   });
-  const [data, retry] = queryResolver.useContinuous<"GetDesiredStates">({
-    kind: "GetDesiredStates",
-    filter,
-    pageSize,
-    currentPage,
-  });
+
   const [compareSelection, setCompareSelection] = useState<CompareSelection>({
     kind: "None",
   });
 
-  function setDeleteModal(version: ParsedNumber, modalState: boolean) {
-    setIsModalOpened(modalState);
-    setVersionToDelete(modalState ? version : 0);
+  const { data, refetch, isError, error, isSuccess } = useGetDesiredStates(
+    envId,
+  ).useContinuous(pageSize, filter, currentPage);
+
+  function setDeleteModal(version: ParsedNumber) {
```

Comment:
why did you need the function key word instead of declaring it as a const?

Also, docstrings

Reply:
changed, added docstrings